### ISSUE TITLE
Aria-Current invalid issue

### ISSUE THREAD

**Issue by aparnapasi** [dequelabs/axe-core#1773]
Aria-current value of slide is being shown as an issue by axe. As per the spec, it seems to be fine. Here is the spec - https://www.w3.org/TR/wai-aria-1.1/#aria-current

With slide value, axe is throwing a violation which should not be the case.

**Comment by straker** [comment]
Do you have an example or code I could look at?

**Comment by WilcoFiers** [comment]
Lets do some testing in AT to make sure you can put aria-current=whatever on an element and still have it be announced as the current element. If that's the case, we can mark this for review.

**Comment by straker** [comment]
Results from screen readers:

```html
<ul>
  <li><a href="foo" aria-current="fllalal">Hi</a></li>
  <li><a href="foo">Hi</a></li>
  <li><a href="foo">Hi</a></li>
</ul>
```

* NVDA / Friefox - "Hi, current link"
* JAWS / IE11 - "Hi, link current"
* VoiceOver / Safar - "current item, link, HI"
* TalkBack / Android - doesn't read aria-current

**Comment by straker** [comment]
@jeankaplansky shouldn't be any doc changes

**Comment by padmavemulapati** [comment]
Ran through code base, as i am not seeing the expected behavour in the latest agora build. Yes, aria-current - issue is in needs-review.
